### ISSUE TITLE
maf2maf: retain-cols doesn't work when the input TSV uses VCF-style indels

### ISSUE THREAD

**Issue by ckandoth** [mskcc/vcf2maf#71]
Reported by @sripakpa. Tested on commit 3f757be

Sample input file with INDELs:
https://drive.google.com/open?id=0B0-e6kHKPsMDaTdRTVNJdTBfMms

Output from maf2maf that is missing columns that were supposed to be retained by `--retain-cols`:
https://drive.google.com/open?id=0B0-e6kHKPsMDLW9vR3dUNHdLYzg

**Comment by ckandoth** [comment]
It also seems like dbSNP IDs are not populated in column 14 and 15 when maf2maf is used.

**Comment by ckandoth** [comment]
I was able to reproduce the problem. I added another column named `Retain_Me` with random data to your sample input file, that I could use with `--retain-cols`. Note that `variant_id` is now a reserved column name that cannot be overridden using `--retain-cols`. It will be borrowed from the `ID` of an input VCF when using vcf2maf. Or from a column named `variant_id` (case insensitive) in an input TSV to maf2maf. So it should work fine here, without needing to specify it in `--retain-cols`.

So here's the input:
```
Chromosome	Start_Position	Reference_Allele	Tumor_Seq_Allele2	Variant_ID	Tumor_Sample_Barcode	Retain_Me
1	565419	C	G	12345	TUMOR	Mango
1	567239	CG	C	12346	TUMOR	Orange
1	761752	C	T	12347	TUMOR	Potato
1	761957	A	AT	12348	TUMOR	Potahto
1	762273	G	A	12349	TUMOR	Lactose
```
And then I ran this:
```
perl maf2maf.pl --input-maf test.NA12878.input.psuedo.maf --output-maf test.NA12878.input.psuedo.vep.maf --retain-cols Retain_Me
```
The resulting MAF properly reformats the VCF-style indels into MAF-style indels, but the data in `Retain_Me` is missing for those indels. The data in `variant_id` is retained because it's handled by different code:
```
Chromosome      Start_Position  End_Position    Reference_Allele        Tumor_Seq_Allele2       variant_id      Retain_Me
1       565419  565419  C       G       12345   Mango
1       567240  567240  G       -       12346
1       761752  761752  C       T       12347   Potato
1       761957  761958  -       T       12348
1       762273  762273  G       A       12349   Lactose
```

**Comment by ckandoth** [comment]
Now fixed in 7689356. I'll run some more tests and tag it as release `v1.6.8` shortly.
